File: roteiro_03/java_logic/desafios/desafio1.java
package roteiro_03.java_logic.desafios;

import java.util.ArrayList;

public class desafio1 {
    public static void main(String[] args) {
        System.out.println("---- Listagem de numeros----");

        for (int i = 1; i <= 10; i++) { 
            System.out.println(""+ i);
        }

        System.out.println("----Listagem de nomes----");
        String[] nomes = {"fernanda","julia","guilherme","davi","rodolfo"};
        
        for (String nome : nomes) {
            System.out.println("" + nome);
        }
        
    }
    
}

File: roteiro_03/cpp_logic/desafio1.cpp
#include <iostream> 
#include <vector>   
#include <string>   

using namespace std;

int main() {

    cout << "--- Contando de 1 a 10 ---" << endl;

    for (int i = 1; i <= 10; i++) {
        cout << i << endl;
    }

    cout << "\n--- Lista de Nomes ---" << endl;

    vector<string> nomes = {"Guilherme", "Julia", "Fernanda", "Davi", "Emily"};


    for (const string& nome : nomes) {
        cout << nome << endl;
    }

    return 0; 
}